src/routes/login/+page.svelte

```diff
@@ -57,7 +59,12 @@
                 },
               );
             } else {
-              goto(redirectBuilder(authResult.user), { replaceState: true });
+              if (authResult.additionalUserInfo.isNewUser) {
+                gtag('event', 'new_account');
+                goto('/account?action=choosePlan');
```

Comment:
I was wondering why when I click "upgrade" and make a new account it takes me the choose plan screen. This doesn't seem intended.